compiler/tflchef/proto/tflchef.proto

```diff
@@ -22,6 +22,7 @@ enum TensorType {
   STRING = 5;
   BOOL = 6;
   INT16 = 7;
+  INT8 = 8;
```

Comment:
circle schema uses `INT8 = 9,` can you use same `9`  ?

Reply:
Ah, I missed the comment. I've updated the code. Thanks!

```
// This enum value corresponds to TensorType in TensorFlow Lite schema
```